tensorflow/core/kernels/fixed_length_record_reader_op.cc

```diff
@@ -107,11 +142,15 @@ class FixedLengthRecordReader : public ReaderBase {
   const int64 record_bytes_;
   const int64 footer_bytes_;
   const int64 hop_bytes_;
+  string lookahead_cache_;
```

Comment:
May be add a comment explaining
1. Why this is needed.
2. What the maximum size of this buffer will be.